kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -442,18 +452,21 @@ private String getClusterKey(Map<String, Object> attributes) {
     }
 
     private Map<String, Object> doDeserialize(SessionInfo sessionInfo,
-            String sessionId) throws IOException, ClassNotFoundException {
+            String sessionId) throws Exception {
         byte[] data = sessionInfo.getData();
         long start = System.currentTimeMillis();
 
         // Is this needed?
         ClassLoader contextLoader = Thread.currentThread()
                 .getContextClassLoader();
         ByteArrayInputStream in = new ByteArrayInputStream(data);
-        Map<String, Object> attributes;
+        Map<String, Object> attributes = new HashMap<>();
```

Comment:
Probably no need to instantiate the map here

Reply:
Thanks for noticing this. Removed the instantiation.